postgres/init.sls

```diff
@@ -126,6 +147,7 @@ postgresql-db-{{ name }}:
     - lc_ctype: {{ db.get('lc_ctype', 'en_US.UTF8') }}
     - lc_collate: {{ db.get('lc_collate', 'en_US.UTF8') }}
     - template: {{ db.get('template', 'template0') }}
+    - tablespace: {{ db.get('tablespace', 'pg_default') }}
```

Comment:
Consider this *related* PR as well: 
https://github.com/saltstack-formulas/postgres-formula/pull/109

There is always going to be a disconnect between allowing users to create/drop/modify databases, users, and - yes - table spaces.  However, it is an *error* to specify a database with a tablespace that doesn't exist, therefore there needs to be a dependency in that direction. There doesn't need to be one in the reverse (`require_in`), as you note, however, and in that case you are right.

At the end of the day, this is easy to fix by placing a requires on the *database* side on the tablespace.